documentation/source/overview/concepts/pam50.rst

```diff
@@ -0,0 +1,84 @@
+.. _pam50:
+
+PAM50 Template
+**************
+
+Introduction
+============
+
+Template-based analysis of multi-parametric MRI data of the spinal cord sets
+the foundation for standardization and reproducibility. Particularly, it
+allows researchers to process their data with minimum bias, perform
+multi-center studies, with the goal of improving patient diagnosis and
+prognosis and helping the discovery of new biomarkers of spinal-related
+diseases.
+
+PAM50 is one such template for MRI of the full spinal cord and brainstem, and
+is included in your installation of SCT. It has the following features:
+
+* It is available for T1-, T2-and T2*-weighted MRI contrast.
+* It is compatible with the ICBM152 brain template, allowing cerebrospinal studies.
+* It includes atlases of white matter pathways and gray matter subregions.
+
+
+Template (``PAM50/template``)
+=============================
+
+The template folder contains image files representing different template diffusion weightings, binary/probablistic masks (for the spinal cord, WM/GM tracts, etc.), and point-wise labels for vertebral levels and intervertebral discs.
+
+The template folder also contains an ``info_label.txt`` file to explain what each file represents:
+
+.. include:: info_label-template.txt
+   :start-line: 3
+   :code:
+
+
+White matter atlas (``PAM50/atlas``)
+====================================
```

Comment:
On closer look, I think the old title here is outdated. This is no longer just a "White matter atlas" now that GM regions have been added as well. 

Possibly this should be named "White and grey matter atlas"

Reply:
EXCELLENT catch! yes, you're absolutely right-- i wanted to add a comment about that yesterday and then got caught into urgent stuff... "White and grey matter atlas" is great 👍